src/jit/gentree.h

```diff
@@ -149,6 +149,18 @@ struct InlineCandidateInfo;
 
 typedef unsigned short AssertionIndex;
 
+//------------------------------------------------------------------------
+// GetAssertionIndex: return 1-based AssertionIndex from 0-based int index.
```

Comment:
@briansull Couldn't we change AssertionProp to stop doing this +1/-1 dance everywhere, and simply burn bit 0, as I've suggested here?